src/mscorlib/src/System/Span.cs

```diff
@@ -74,13 +74,18 @@ private Span(ref T ptr, int length)
 
         public int Length
         {
-            get
-            {
-                return _length;
-            }
+            get { return _length; }
         }
 
-        public static Span<T> Empty { get { return default(Span<T>); } }
+        internal long SizeInBytes
```

Comment:
It would be better for this to return `nuint`. `long` will have unnecessary overhead on 32-bit platforms.